TCMSApp/src/client/app/tests/tests-list.controller.js

```diff
@@ -0,0 +1,63 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('app.tests')
+        .controller('TestsListController', TestsListController);
+
+    TestsListController.$inject = ['logger', 'FakeTestsFactory', '$uibModal', 'TestsService'];
+    /* @ngInject */
+    function TestsListController(logger, FakeTestsFactory, $uibModal, TestsService) {
+        var vm = this;
+
+        activate();
+
+        vm.checkedAllCases = false; // checkbox for testCases
+        vm.openAddSuite = openAddSuite; // open modal for new suite
+        vm.f = TestsService.getFakeSuites();// f for faker, array of suites
+        vm.currentSuite = vm.f[0]; 
```

Comment:
and the last one.

*nerd mode on*

you can see that you store `currentSuite` in 2 different places (here and in TestsService). I suggest you to store it (and determine) only in TestsService and here have the reference: 
`vm.currentSuite = TestsService.getCurrentSuite(vm.currentSuite);`

how do you think? :)